createBetaMap.py

```diff
@@ -12,9 +12,9 @@
 # Add the common include directory before importing our custom libraries
 sys.path.append("include")
 
-from ascFile import *
-from calibrationLib import *
-from utility import *
+from include.ascFile import *
+from include.calibrationLib import *
+from include.utility import *
```

Comment:
Why is there an `include` prefix here? The `sys.path.append` should render this unnecessary.

Reply:
The command line is not able to recognize the `utility`, `ascFile`, `CalibrationLib`. It says file not found. But while running directly from PyCharm, it works well (without using include).